### PR TITLE
Added script to apply all new evolutions

### LINKED ISSUE #3423: script to apply missing migrations
could be called as yarn command

compare current schema version with evolutions present in conf directory, apply as needed?

(what about diverging branches, though? let’s for now only implement the forward direction)

### PR DESCRIPTION
**Pull request by rschwanhold**
### Steps to test:
- apply the last reversion
- run `yarn apply-evolutions`

### Issues:
- fixes #3423 

------
- [ ] ~~Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~~
- [ ] ~~Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~
- [ ] ~~Updated [documentation](../blob/master/docs) if applicable~~
- [ ] ~~Needs datastore update after deployment~~
- [x] Ready for review

### CODE REVIEW

File: tools/postgres/apply_evolutions.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+# assert that the found schema version is a number
+re='^[0-9]+$'
+if ! [[ $schema_version =~ $re ]] ; then
+   echo "Error: Schema versoin is not a number" >&2; exit 1
+fi
+
+# get list of schemas to apply 
+files=""
+for entry in "../../conf/evolutions"/*
```
Comment (rschwanhold) [inline tools/postgres/apply_evolutions.sh]: How can I get the absolute path to this folder?
When I run `./apply_evolutions.sh` from the right directory everything works, but when I run `yarn apply-evolutions` it seems like it does not find the evolutions.

Comment (fm3) [reply tools/postgres/apply_evolutions.sh]: you could make the path be relative to the script’s directory
https://stackoverflow.com/a/246128/7564727

Comment (jstriebel) [reply tools/postgres/apply_evolutions.sh]: We usually use `scriptdir="$(dirname "$0")"` to get the path of the script, and the use everything relatively to that. An example is https://github.com/scalableminds/webknossos/blob/master/tools/postgres/ensure_db.sh

File: tools/postgres/apply_evolutions.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+# assert that the found schema version is a number
+re='^[0-9]+$'
+if ! [[ $schema_version =~ $re ]] ; then
+   echo "Error: Schema versoin is not a number" >&2; exit 1
+fi
+
+# get list of schemas to apply 
+files=""
+for entry in "${scriptdir}/../../conf/evolutions"/*
```
Comment (jstriebel) [inline tools/postgres/apply_evolutions.sh]: You can simply do `for entry in …/*.sql`, and remove the following `if` check.

**Pull request merged**